cypress/page/shoping-cart.page.ts

```diff
@@ -0,0 +1,19 @@
+class ShopingCartPage {
+    private checkoutPopUpBtn: string;
+
+    private checkoutSummaryBtn: string;
+
+    constructor() {
+      this.checkoutPopUpBtn = "[style*=\"display: block;\"] .button-container > a";
+      this.checkoutSummaryBtn = ".cart_navigation span";
+    }
+
+    public checkoutPopUpEvent(): void {
```

Comment:
the methods on page objects generally should be called with the name of business actions, without referring for example to the element or events like in this case PopUpEvent... instead of that, you can call it like addToCart or selectTShirt